client/src/PublicRequest.ts

```diff
@@ -93,16 +93,19 @@ export type ExportResult = {
     wordsExported: boolean,
 };
 
-export type SignTransactionRequestSimple = SimpleRequest & TransactionInfo & {
-    layout?: Exclude<SignTransactionRequestLayout, 'checkout'>,
+type SignTransactionRequestCommon = SimpleRequest & TransactionInfo & {
```

Comment:
👍

Reply:
What do you think about `SignTransactionRequestBase`?

Reply:
`Comon` is good in my opinion.